app.py

```diff
@@ -95,23 +133,23 @@ async def create_port_webhook():
         },
         "integrationType": "custom",
     }
-
-    try:
-        response = await client.post(
-            f"{PORT_API_URL}/webhooks",
-            json=webhook_data,
-            headers=port_headers,
-        )
-        response.raise_for_status()
+    response = await send_port_request(
+        "POST",
+        f"webhooks",
+        webhook_data
+    )
+    if isinstance(response, dict):
+        status_code = response.get("status_code")
+        if status_code == 442:
+            logger.error("Incorrect mapping, kindly fix!")
+            return None
+        else:
+            logger.error(f"Error creating Port webhook: {status_code}")
+            return None
```

Comment:
let's log the actual error stack trace. the status code may not be helpful at times

Reply:
Done